draft-gpew-priv-ppm.md

```diff
@@ -205,10 +205,11 @@ schemes that implement the VDAF interface specified in
   etc. This class of VDAFs is based on Prio [CGB17] and includes improvements
   described in [BBCGGI19].
 
-* `hits`, which allows for finding the most common strings among a collection
+* `pops`, which allows for finding the most popular strings among a collection
```

Comment:
As @bifurcation noted in https://github.com/cjpatton/vdaf/pull/16, I think `poplar` is nicer.

Reply:
"Poplar" is the name of the full-blown protocol, whereas this name is supposed to refer to the underlying VDAF. I'd like to avoid conflating the two.

Reply:
I guess we're just voting now, but I would prefer poplar.

Reply:
Well then, this just got interesting. I wonder if others have an opinion or ideas for alternatives. I don't care much about the name for the VDAF, my only criterion is that it is distinct from "Poplar". A more verbose possibility is `poplar-vdaf`. 
cc/ @csharrison @eriktaubeneck @martinthomson @stpeter @bifurcation @BranLwyd

Reply:
You could align with the naming scheme for the Prio VDAF and call it `poplar1` :) Other than that I don't have a strong opinion.

Reply:
Keeping it consistent with the naming scheme for Prio seems positive, and a version number doesn't seem like a terrible addition as it seems likely the VDAF may evolve like the prio VDAF did.

Reply:
I like your thinking @eriktaubeneck  and @csharrison ! I vote for `poplar1`.

Reply:
s/pops/poplar1/.